migration/src/m0000030_create_sbom.rs

```diff
@@ -26,11 +26,11 @@ impl MigrationTrait for Migration {
                     .if_not_exists()
                     .col(ColumnDef::new(Sbom::SbomId).uuid().not_null())
                     .col(ColumnDef::new(Sbom::NodeId).string().not_null())
-                    .col(ColumnDef::new(Sbom::Location).string().not_null())
                     .col(ColumnDef::new(Sbom::DocumentId).string().not_null())
                     .col(ColumnDef::new(Sbom::Sha256).string().not_null())
                     .col(ColumnDef::new(Sbom::Published).timestamp_with_time_zone())
                     .col(ColumnDef::new(Sbom::Authors).array(ColumnType::String(None)))
+                    .col(ColumnDef::new(Sbom::Labels).json_binary().not_null())
```

Comment:
I think we're squarely in the phase of "no changes to existing migrations if possible". Instead of changing two migrations, can you create a new one with the necessary column changes to both tables?